OpenSim/Simulation/Model/ScapulothoracicJoint.h

```diff
@@ -0,0 +1,183 @@
+#ifndef OPENSIM_SCAPULOTHORACIC_JOINT_H_
+#define OPENSIM_SCAPULOTHORACIC_JOINT_H_
+/* -------------------------------------------------------------------------- *
+ *                     OpenSim:  ScapulothoracicJoint.h                       *
+ * -------------------------------------------------------------------------- *
+ * ScapulothoracicJoint is offered as an addition to the OpenSim API          *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+* Copyright (c) 2005-2017 Stanford University and the Authors                *
+ * Author(s): Ajay Seth                                                       *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+// INCLUDE
+#include <OpenSim/Simulation/SimbodyEngine/Joint.h>
+
+namespace OpenSim {
+
+/**
+ * A class implementing a 4-DOF ScapulothoracicJoint.
+ *
+ * Motion of the scapula is described by an ellipsoid surface fixed to the
+ * thorax upon which the joint frame of scapul rides. The DOFs are:
+ *
+ * - 2 DOFs: up-down/medio-laternal: The motion on the surface. These are
+ *           described by latitude and longitudinal angles.
+ *
+ * - 1 DOF:  scapula rotation about the normal to the ellipsoid surface
+ *
+ * - 1 DOF:  Rotation about a "winging" axis defined by a point and axis
```

Comment:
winging: rotation about and axis fixed to the scapula frame and defaults to scapula-y